auth0_flutter_platform_interface/lib/src/method_channel_auth0_flutter_auth.dart

```diff
@@ -15,6 +16,7 @@ import 'user_profile.dart';
 
 const MethodChannel _channel = MethodChannel('auth0.com/auth0_flutter/auth');
 const String authLoginMethod = 'auth#login';
+const String authLoginWithOtpMethod = 'auth#login_otp';
```

Comment:
All the other method keys use camelCase, please use it for this new key as well.